providers/onshape/onshape_provider/sketch.py

```diff
@@ -184,16 +186,32 @@ def create_from_vertices(
         raise NotImplementedError()
 
     def create_point(self, point: PointOrListOfFloatOrItsStringValue) -> "Vertex":
-        raise NotImplementedError()
+        point = Point.from_list_of_float_or_string(point)
+        api_resp = onshape_actions.create_point(self.client, self.onshape_url, self.name, point)
+        json_native_data = json.loads(api_resp.data)
+        return Vertex(location=point, native_instance=json_native_data["feature"]["entities"][0])
 
     def create_line(
         self,
         start_at: PointOrListOfFloatOrItsStringValueOrVertex,
         end_at: PointOrListOfFloatOrItsStringValueOrVertex,
     ) -> "Edge":
+        start_point = Point.from_list_of_float_or_string_or_Vertex(start_at)
+        end_point = Point.from_list_of_float_or_string_or_Vertex(end_at)
+        api_resp = onshape_actions.create_line(self.client, self.onshape_url,self.name, start_point, end_point)
+        json_native_data = json.loads(api_resp.data)        
+        return Edge(
+            v1=None,
+            v2=None,
+            name="",
+            native_instance=json_native_data["feature"]["entities"][0]
+            )
         raise NotImplementedError()
```

Comment:
This line is never reached, please remove it. There are a few more like this, if you could get them that would be great. Thank you.